src/main/java/org/takes/facets/auth/social/PsGoogle.java

```diff
@@ -143,9 +143,9 @@ public Response exit(final Response response,
      */
     private Identity fetch(final String token) throws IOException {
         // @checkstyle LineLength (1 line)
-        final String uri = new Href(this.gapi).path("oauth2").path("v1")
-            .path("userinfo")
-            .with("alt", "json")
+        final String uri = new Href(this.gapi).path("plus").path("v1")
+            .path("people")
+            .path("me")
             .with("access_token", token)
```

Comment:
@Happy-Neko on the screenshot in the original issue you can see a selector `image/url` in parameters
will the api call work without this parameter?
did you try this api call on real server?